src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetVOD/index.spec.tsx

```diff
@@ -0,0 +1,172 @@
+import { cleanup, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { liveState } from 'types/tracks';
+import { videoMockFactory } from 'utils/tests/factories';
+import { wrapInIntlProvider } from 'utils/tests/intl';
+import { DashboardVideoLiveWidgetVOD } from '.';
+
+jest.mock('data/appData', () => ({
+  appData: {
+    jwt: 'json web token',
+  },
+}));
+
+describe('DashboardVideoLiveWidgetVOD', () => {
+  it('renders default widget components', () => {
+    const video = videoMockFactory({
+      live_state: liveState.STOPPED,
+    });
+
+    render(wrapInIntlProvider(<DashboardVideoLiveWidgetVOD video={video} />));
+
+    screen.getByText('VOD');
+    userEvent.click(screen.getByTitle('help'));
+    screen.getByText('This widget allows you to handle live vod features.');
+  });
+
+  it('show harvesting message when live is harvesting', () => {
+    const video = videoMockFactory({
+      live_state: liveState.HARVESTING,
+    });
+
+    render(wrapInIntlProvider(<DashboardVideoLiveWidgetVOD video={video} />));
+
+    screen.getByText('Harvesting in progress...');
+
+    expect(
+      screen.queryByText(/There is nothing to harvest/),
+    ).not.toBeInTheDocument();
+    expect(
+      screen.queryByRole('button', { name: 'Create vod' }),
+    ).not.toBeInTheDocument();
+    expect(
+      screen.queryByRole('button', { name: 'Publish vod' }),
+    ).not.toBeInTheDocument();
+  });
+
+  it('shows message to stop the live if live is not stopped or harvested', () => {
```

Comment:
```suggestion
  it('shows message to stop the live if live is not stopped or harvested', () => {
```
I don't see any message or test for this to confirm there is a message to stop a live

Reply:
```
noHarvestMessage: {
    defaultMessage:
      'There is nothing to harvest. To create your video, you have to record your stream.{br}Harvest will be available at the end of the live.',
    description:
      'Message displays in the VOD widget when the is nothing to harvest.',
    id: 'components.DashboardVideoLiveWidgetVOD.noHarvestMessage',
  }
```

That's the message